tasks/oah-validate.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Validate Installation
+  stat:
+    path: "/opt/spdx-sbom-generator"
+  register: spdx_executable 
+
+- name: Report if file doesn't exist

```

Comment:
Please include a message for when the tool is found

Reply:
Added a message for when tool is found

Reply:
Replaced a debug message with fail statement when tool is not found